.github/workflows/lint-test.yaml

```diff
@@ -143,7 +143,10 @@ jobs:
           fetch-depth: 0 # get all the history because cargo xtest --change-since origin/main requires it.
       - uses: ./.github/actions/rust-setup
       - run: docker run --detach -p 5432:5432 cimg/postgres:14.2
-      - run: cargo nextest --nextest-profile ci --unit --exclude backup-cli --changed-since "origin/main"
+      - uses: taiki-e/install-action@v1.5.6
+        with:
+          tool: nextest
+      - run: cargo nextest run --profile ci --workspace --exclude smoke-test --exclude backup-cli --exclude testcases
```

Comment:
this seems different? missing --unit and adding --exclude testcases?

Reply:
the `--unit` flag was just a fancy `x` specific way of saying "don't run these tests": https://github.com/aptos-labs/aptos-core/blob/main/x.toml#L4-L6 ,  aka exclude `smoke-test` and `testcases`.
see also https://github.com/aptos-labs/aptos-core/blob/main/devtools/x/src/nextest.rs#L130